implementations/rust/ockam/ockam/src/profile/profile_change_history.rs

```diff
@@ -89,7 +89,11 @@ impl ProfileChangeHistory {
             RotateKey(change) => change.data().public_key(),
         };
 
-        Ok(PublicKey::new(data.into()))
```

Comment:
Clippy complained that this function always succeeded, thus did not need to be wrapped in a Result. But all of our APIs should return Result, and we will probably find ways to make it fail in the future :) So I started with a trivial failure mode.

Reply:
Nice